source/tutorials/first-steps/index.md

```diff
@@ -0,0 +1,12 @@
+# First Steps
+
+This tutorial series is meant to be your first interaction with Nix.
```

Comment:
```suggestion
This tutorial series is where you should start learning Nix.
```

Reply:
This probably also needs some brief overview of what you'll learn in this series.